torchao/dtypes/fbgemm_fp8_tensor.py

```diff
@@ -26,7 +26,16 @@
 
 class FbgemmFp8Tensor(TorchAOBaseTensor):
     """
+    Float8 Rowwise Quantized (weight) Tensor, with float8 rowwise dynamic quantization for activation.
     TODO: needs padding for cutlass kernels
+
+    Tensor Attributes:
+        float8_data: float8 raw data, dtype torchao.float8.config.e4m3_dtype
```

Comment:
to clarify, does this mean there is a dependency on `torchao.float8.config.e4m3_dtype`? If so, I think the dependency should be refactored away to a common utility, it's not expected for that config to affect anything other than the `torchao.float8` workflow.